report_github_metrics.rb

```diff
@@ -21,7 +21,7 @@ def collect_job_metrics(job, tags)
 def collect_workflow_metrics(jobs, tags)
   start = jobs.min_by{|job| job["started_at"]}["started_at"]
   finish = jobs.max_by{|job| job["completed_at"]}["completed_at"]
-  status = jobs.all?{|job| job["conclusion"] == "success"} ? "success" : "failure"
+  status = jobs.all?{|job| ["success", "skipped"].include? job["conclusion"]} ? "success" : "failure"
```

Comment:
Any benefit in tracking "skipped" status separately?

Reply:
I thin workflows would have at least one job run, so I'm not sure we would ever get a fully skipped workflow.

Reply:
👍  Makes sense.